bindata/network/multus/multus.yaml

```diff
@@ -455,3 +455,42 @@ spec:
           configMap:
             name: cni-binary-copy-script
             defaultMode: 0744
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: ip-reconciler
+  namespace: openshift-multus
+  labels:
+    tier: node
+    app: whereabouts
+spec:
+  schedule: "*/5 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          priorityClassName: "system-node-critical"
+          securityContext:
+            privileged: true
```

Comment:
`privileged` is only defined in the container-level `securityContext`, not the pod-level. So this has no effect